racecar_beacon/package.xml

```diff
@@ -2,23 +2,25 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>racecar_beacon</name>
-  <version>0.0.0</version>
-  <description>TODO: Package description</description>
-  <maintainer email="lali3401@usherbrooke.ca">racecar</maintainer>
+  <version>2025.4.12</version>
+  <description>Package for GRO520-related features</description>
+  <maintainer email="fecm0701@usherbrooke.ca">Marc-Olivier Fecteau</maintainer>
   <license>Apache-2.0</license>
 
+  <buildtool_depend>ament_python</buildtool_depend>
+
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>nav_msgs</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
+  <exec_depend>tf_transformations</exec_depend>
```

Comment:
I know this is not your change but since we thouching this can you sort the dependecy in alphabetic other